src/Nethermind/Nethermind.Consensus.AuRa/Contracts/Contract.ConstantContract.cs

```diff
@@ -43,46 +40,38 @@ public ConstantContract(Contract contract, IReadOnlyTransactionProcessorSource r
                 _readOnlyReadOnlyTransactionProcessorSource = readOnlyReadOnlyTransactionProcessorSource ?? throw new ArgumentNullException(nameof(readOnlyReadOnlyTransactionProcessorSource));
             }
         
-            public byte[] Call(BlockHeader header, Transaction transaction)
+            public byte[] Call(BlockHeader parentHeader, Transaction transaction)
```

Comment:
can you add comments to public methods on this abstract class? will help people use it in the future